README.md

```diff
@@ -70,10 +72,12 @@ The user can specify the flow velocity vector (in m/s) and the domain geometry (
 Run the simulation:
 
 ```python
-output = scenario.simulate(object_path="vehicle.obj",
+task = scenario.simulate(object_path="vehicle.obj",
```

Comment:
In theory, yes. But I am digging into it and will solve it when I have a final answer.

No, for gromacs is (ns) and the for all the others in (s).

Reply:
OK. In that case I think we should assume that

```
simulation_time 
```
is always in seconds but if there is another unit then we need to add a suffix to it:

```
simulation_time_ns
```
otherwise it is confusing for the user.

Reply:
@cristianacarpinteiro @david451lima : wdyt of my comment above?

Reply:
I don't see a big problem in `simulation_time` having different units in different scenarios. Users of one scenario (or one type, say MD) will not likely use others (say fluid dynamics).

Also, I think this opens a precedent for us naming distance parameters differently too, e.g. `height`, `height_km`, `height_fm`. The main challenge is then to find the "default" units for all methods across the API. We could use SI, but sometimes the natural units are different in different problems.

Reply:
I see your point. Let's do it, then 💪

Reply:
The actual simulation time used in the simulator is in (fs). But it is annoying for the user to say 10 000 000 fs when inserting the time.

But I agree with @sarmento on this one, at least for the time, even though I like the approach also for the height.
I will set `simulation_time_ns` for now and open an issue on it!

Reply:
For gromacs the most natural time frame is nanoseconds, microseconds would be a really long simulation. Most of the studies do simulaitons around 200 ns. But I understand your point! Let's open an issue about it.